main.tex

```diff
@@ -0,0 +1,175 @@
+
+\subsection{Decision Variables}
+
+\begin{itemize}
+    \item \textbf{Route Decision Variables:} A binary decision variable was created for each route, showing whether a particular supply-to-demand route is chosen.
+\end{itemize}
+
+\subsection{Objective Function}
+
+The objective function is to minimize the total transportation cost, which is the sum of the transportation costs associated with each selected route. The objective function is given as:
+
+\[
+\text{Minimize } Z = \sum_{w \in W, b \in B} \text{cost}_{wb} \cdot x_{wb}
+\]
+
+Where:
+\begin{itemize}
+    \item \( \text{cost}_{wb} \) is the cost of transporting products from warehouse \( w \) to demand node \( b \),
+    \item \( x_{wb} \) is a binary variable indicating whether the route from warehouse \( w \) to demand node \( b \) is selected.
+\end{itemize}
+
+\subsection{Constraints}
+
+The optimization model is subject to several constraints:
+
+\begin{figure}[!h]
+     \begin{center}
+     \includegraphics[width=0.6\paperwidth]{route.png}
+          \end{center}
+     \caption{Route constraints}
+     \label{fig:cd}
+\end{figure}
+
+
+\begin{itemize}
+    \item \textbf{Supply Constraints:} The total quantity shipped from a warehouse cannot exceed its supply capacity.
+    \item \textbf{Demand Constraints:} The total quantity received at a demand node must meet or exceed its demand.
+    \item \textbf{Self-Transportation Constraints:} Routes from a supply node to itself were excluded.
+\end{itemize}
+
+\section{Results}
+
+
+
+The optimization was performed using the \texttt{Pulp} Python library, which solved the MIP problem using the default solver. The solution provided the optimal transportation routes and the associated costs. The following observations could be made:
+
+
+
+\begin{table}[h!]
+    \centering
+    \begin{tabular}{|c|c|}
+        \hline
+        \textbf{Route} & \textbf{Value} \\
+        \hline
+        Route\_PLANT01\_PORT09 & 1070.0 \\
+        Route\_PLANT02\_PORT09 & 138.0 \\
+        Route\_PLANT03\_PORT09 & 555600470.0 \\
+        Route\_PLANT04\_PORT09 & 554.0 \\
+        Route\_PLANT05\_PORT09 & 385.0 \\
+        Route\_PLANT06\_PORT09 & 49.0 \\
+        Route\_PLANT07\_PORT09 & 265.0 \\
+        Route\_PLANT08\_PORT09 & 14.0 \\
+        Route\_PLANT09\_PORT09 & 11.0 \\
+        Route\_PLANT10\_PORT09 & 118.0 \\
+        Route\_PLANT11\_PORT09 & 332.0 \\
+        Route\_PLANT12\_PORT09 & 209.0 \\
+        Route\_PLANT13\_PORT09 & 490.0 \\
+        Route\_PLANT14\_PORT09 & 549.0 \\
+        Route\_PLANT15\_PORT09 & 11.0 \\
+        Route\_PLANT16\_PORT09 & 457.0 \\
+        Route\_PLANT17\_PORT09 & 8.0 \\
+        Route\_PLANT18\_PORT09 & 111.0 \\
+        Route\_PLANT19\_PORT09 & 7.0 \\
+        \hline
+    \end{tabular}
+    \caption{Route Values for PORT09}
+\end{table}
```

Comment:
The table presents route values but could be improved for clarity and context:

1. **Visualization**: A bar chart would be a better alternative to compare route values visually, making it easier to identify trends or outliers.

2. **Integer Values**: These values should likely be integers (e.g., quantities or costs). If so, verify the data source to ensure accurate formatting.

3. **Why Only PORT09?**: It's unclear why the table focuses exclusively on PORT09. Was this constraint introduced during data preparation or filtering? If so, it should be explicitly mentioned and justified earlier in the report.

4. **Context**: The table lacks an explanation of what "Value" represents (e.g., quantities, costs, or another metric). Adding context and units would improve interpretability.

This table could be significantly improved by addressing these issues and providing more clarity about the constraint and the data filtering process.